chapter_preliminaries/calculus.md

```diff
@@ -257,11 +257,11 @@ $$\nabla_{\mathbf{x}} f(\mathbf{x}) = \bigg[\frac{\partial f(\mathbf{x})}{\parti
 
 where $\nabla_{\mathbf{x}} f(\mathbf{x})$ is often replaced by $\nabla f(\mathbf{x})$ when there is no ambiguity.
 
-Let $\mathbf{A}$ be a matrix with $m$ rows and $n$ columns, and $\mathbf{x}$ be an $n$-dimensional vector, the following rules are often used when differentiating multivariate functions:
+Let $\mathbf{x}$ be an $n$-dimensional vector, the following rules are often used when differentiating multivariate functions:
 
-* $\nabla_{\mathbf{x}} \mathbf{A} \mathbf{x} = \mathbf{A}^\top$,
-* $\nabla_{\mathbf{x}} \mathbf{x}^\top \mathbf{A}  = \mathbf{A}$,
-* $\nabla_{\mathbf{x}} \mathbf{x}^\top \mathbf{A} \mathbf{x}  = (\mathbf{A} + \mathbf{A}^\top)\mathbf{x}$,
+* If $\mathbf{A} \in \mathbb{R}^{m \times n}$, then $\nabla_{\mathbf{x}} \mathbf{A} \mathbf{x} = \mathbf{A}^\top$,
+* If $\mathbf{A} \in \mathbb{R}^{n \times m}$, then $\nabla_{\mathbf{x}} \mathbf{x}^\top \mathbf{A}  = \mathbf{A}$,
+* If $\mathbf{A} \in \mathbb{R}^{n \times n}$, then $\nabla_{\mathbf{x}} \mathbf{x}^\top \mathbf{A} \mathbf{x}  = (\mathbf{A} + \mathbf{A}^\top)\mathbf{x}$,
```

Comment:
Thanks! My suggestions:
If XXX, then YYY -> For all XXX, YYY.

Reply:
Change to following:
![image](https://user-images.githubusercontent.com/37914843/69767143-b79ee380-112f-11ea-870e-29e25a4039dd.png)